git_aula.md

```diff
@@ -0,0 +1 @@
+content is not important
```

Comment:
@raplevy Not a bad start! Maybe try some markdown? Try to make something **bold** or *italic* or a [link](https://guides.github.com/features/mastering-markdown/)